src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/EvaluateConcordanceOnNonGenotypedSVVCFs.java

```diff
@@ -0,0 +1,497 @@
+    protected TriPredicate<VariantContext, ReferenceContext> getConcordanceTester() {
+        return (eval, truth, r) -> recipOvpFrac <= computeConcordanceScore(eval, truth, r);
+    }
+
+    /**
+     * >= 0.5 is considered concordant.
+     */
+    @Override
+    protected float computeConcordanceScore(final VariantContext eval, final VariantContext truth, final ReferenceContext context) {
+        final String evalType = eval.getAttributeAsString(GATKSVVCFConstants.SVTYPE, "");
+        final String truthType = truth.getAttributeAsString(GATKSVVCFConstants.SVTYPE, "");
+
+        // TODO: 6/3/18 we only consider insertion length but DO NOT YET MATCH inserted sequence (we should, in the long run, but that involves haplotype reconstruction)
+        if (evalType.equals(SimpleSVType.SupportedType.INS.name()) || evalType.equals(SimpleSVType.SupportedType.DUP.name())) {
+            if (truthType.equals(SimpleSVType.SupportedType.INS.name())) { // TODO: 6/3/18 truth type doesn't call DUP, instead type them as INS
+                int evalLen = eval.getAttributeAsInt(GATKSVVCFConstants.SVLEN, 0);
+                int truthLen = truth.getAttributeAsInt(GATKSVVCFConstants.SVLEN, 0);
+                float diff = Math.abs(evalLen - truthLen);
+                return 1 - insLenDiffScoreCoeff * Math.max(diff/evalLen, diff/truthLen);
+            } else
+                return 0f;
+        } else if (evalType.equals(SimpleSVType.SupportedType.DEL.name())){
+
+            if (truthType.equals(SimpleSVType.SupportedType.DEL.name())) { // NOTE: one can use SVLEN, but remember the truth set might report positive SVLEN for DEL
+                SVInterval evalRange = new SVInterval(0, eval.getStart() - 1, eval.getEnd());
+                SVInterval truthRange = new SVInterval(0, truth.getStart() - 1, truth.getEnd());
+                float overlapLen = evalRange.overlapLen(truthRange);
+                return Math.min(overlapLen/evalRange.getLength(), overlapLen/truthRange.getLength());
+            } else if (truthType.equals(SimpleSVType.SupportedType.INS.name()) && eval.hasAttribute(GATKSVVCFConstants.INSERTED_SEQUENCE_LENGTH)) { // some call set might report replacement as deletion with insertion
+                final int truthInsLen = truth.getAttributeAsInt(GATKSVVCFConstants.SVLEN, 0);
+                final int evalInsLen = eval.getAttributeAsInt(GATKSVVCFConstants.INSERTED_SEQUENCE_LENGTH, 0);
+                float diff = Math.abs(evalInsLen - truthInsLen);
+                return 1 - insLenDiffScoreCoeff * Math.max(diff/evalInsLen, diff/truthInsLen);
+            } else {
+                return 0f;
+            }
+        } else if (evalType.equals(SimpleSVType.SupportedType.INV.name())) {
+
+            if (evalType.equalsIgnoreCase(truthType)) { // NOTE: one can use SVLEN, but remember the truth set might report SVLEN as positive or 0 for INV
+                SVInterval evalRange = new SVInterval(0, eval.getStart() - 1, eval.getEnd());
```

Comment:
Please review the position number used for SVInterval everywhere in this PR.

SimpleInterval: S...E (biologist preference)
BED: (S-1)...E (computer scientist preference)
SVInterval: S...(E+1) (optimized for GATK-SV purposes)

Reply:
What do you mean by review? 
I've been using `SVInterval` with [0, end) convention for all my code, i.e. following the BED coordinate.

Reply:
I see what you mean. It turns out that I've been using it in the un-intended way.
Well, I guess now I have to deal with another set of coordinate.